src/utils/internalState.ts

```diff
@@ -0,0 +1,18 @@
+import { ContextProps } from './formContext';
+
+type InternalState = {
+  formState: {
+    [formUuid: string]: { contextProps: null | ContextProps; currentStep: any };
+  };
+};
+
+const internalState: InternalState = {
+  formState: {}
+};
+
+// do we want this fn? mostly did it to be semi-consistent with the initState
+function getInternalState() {
```

Comment:
do we want this fn? mostly did it to be semi-consistent with the initState

Reply:
i don't think we need this function. initState function is used because it also contains validation logic that runs